src/components/layout/header/helpers.ts

```diff
@@ -0,0 +1,48 @@
+export function getLinks({
```

Comment:
Nenapíšeme to přes flat arrow? :)
`export const getLinks  = (...) => `

Reply:
Jsem zvyklý psát spíše pojmenované funkce z důvodu čitelnějších chybových hlášek, ale je mi to ve výsledku jedno a je pravdou, že komponenty takto máme a měli bychom být jednotní (rovnou kouknu i po ESLint pravidle). 

Díky. 👍